src/DssSpell.t.sol

```diff
@@ -147,27 +209,39 @@ contract DssSpellTest is DssSpellTestBase {
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        checkChainlogKey("RWA007_A_INPUT_CONDUIT");
-        checkChainlogKey("RWA007_A_JAR_INPUT_CONDUIT");
+        checkChainlogKey("DIRECT_HUB");
+        checkChainlogKey("DIRECT_MOM");
+        checkChainlogKey("DIRECT_MOM_LEGACY");
+
+        checkChainlogKey("DIRECT_COMPV2_DAI_POOL");
+        checkChainlogKey("DIRECT_COMPV2_DAI_PLAN");
+        checkChainlogKey("DIRECT_COMPV2_DAI_ORACLE");
+
+        checkChainlogKey("MCD_CLIP_CALC_GUSD_A");
+        checkChainlogKey("MCD_CLIP_CALC_USDC_A");
+        checkChainlogKey("MCD_CLIP_CALC_PAXUSD_A");
 
-        checkChainlogVersion("1.14.5");
+        checkChainlogKey("STARKNET_GOV_RELAY_LEGACY");
+        checkChainlogKey("STARKNET_GOV_RELAY");
+
+        checkChainlogVersion("1.14.6");
     }
 
-    function testNewIlkRegistryValues() private { // make private to disable
+    function testNewIlkRegistryValues() public { // make private to disable
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
         // Insert new ilk registry values tests here
-        // RETH-A
-        //assertEq(reg.pos("RETH-A"),    54);
-        //assertEq(reg.join("RETH-A"),   addr.addr("MCD_JOIN_RETH_A"));
-        // assertEq(reg.gem("RETH-A"),    addr.addr("RETH"));
-        // assertEq(reg.dec("RETH-A"),    GemAbstract(addr.addr("RETH")).decimals());
-        // assertEq(reg.class("RETH-A"),  1);
-        // assertEq(reg.pip("RETH-A"),    addr.addr("PIP_RETH"));
-        // assertEq(reg.name("RETH-A"),   "Rocket Pool ETH");
-        // assertEq(reg.symbol("RETH-A"), GemAbstract(addr.addr("RETH")).symbol());
+        // DIRECT-COMPV2-DAI
+        assertEq(reg.pos("DIRECT-COMPV2-DAI"),    55);
+        assertEq(reg.join("DIRECT-COMPV2-DAI"),   addr.addr("DIRECT_HUB"));
+        assertEq(reg.gem("DIRECT-COMPV2-DAI"),    address(0));
```

Comment:
Shouldn't this might be `cDai`? haven't checked yet how we did in the aave D3M.

Reply:
Yeah for Aave we completed with the `aDai` data (the gem, decimals, token symbol and name).

Reply:
Added gem and pip recommendations.